### ISSUE TITLE
GlusterFS volume create fails when launching a single control node

### ISSUE THREAD

**Issue by chrholme** [CiscoCloud/microservices-infrastructure#674]
glusterfs_volume.sh.j2 contains the command:
```bash
NAME=$1

if ! gluster volume list | grep -q $NAME; then
    gluster volume create $NAME \
            replica {{ glusterfs_replication }} \
            {% for server in groups[glusterfs_server_group] %}{{ server }}.node.{{ consul_dns_domain }}:{{ glusterfs_brick_location }}/$NAME {% endfor %}{% if glusterfs_volume_force == "yes" %}force{% endif %};
else
    echo "volume already created";
fi;
```
If {{glusterfs_replication}} replication value is 1 in the case of a single control node the volume create command will fail as 1 is not a valid replica count.  

I'm thinking there should be an {% if glusterfs_replication <= 1%} to catch the single controller case and run the volume create command without the replica parameter set.

**Comment by siddharthist** [comment]
@BrianHicks You mentioned in #733 that we may have depreciated single control node setups. Is that correct? If so, we can close this issue.

**Comment by BrianHicks** [comment]
@chrholme we really can't recommend using single-control-node clusters, as data loss is almost guaranteed in any outage, and you have no capacity for maintenance. But we could really stand to have a more friendly error message indicating that in this case. I think that's going to have to be the solution to this ticket. Were you able to get around this? I'm sorry that this issue fell through the cracks.

**Comment by chrholme** [comment]
I fully agree that single node master is a bad idea for anything prod but its nice to have it as an option for spinning up POCs.  That said I'm happy to hack the Ansible for this use case if its not going to be officially supported.

**Comment by siddharthist** [comment]
@crholme I actually have a branch with your suggested fix, I just haven't been able to test yet. I'll keep you posted. If you'd like to try it out and test as well, that's be super helpful.

**Comment by siddharthist** [comment]
@chrholme If you're interested in testing/trying it out, `feature/configurable-vms` should fix the issue.

**Comment by siddharthist** [comment]
Fixed in #942
